services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/DirectionsResponse.java

```diff
@@ -66,16 +66,10 @@ public static Builder builder() {
    * List of {@link DirectionsWaypoint} objects. Each {@code waypoint} is an input coordinate
    * snapped to the road and path network. The {@code waypoint} appear in the list in the order of
    * the input coordinates.
-   * @deprecated the waypoints are returned in the root of the response only if
```

Comment:
Why did you remove `@Deprecated`? It's deprecated on backend.

Reply:
Context in https://github.com/mapbox/mapbox-java/pull/1509#discussion_r1017895480.

Reply:
https://github.com/mapbox/mapbox-java/pull/1509#discussion_r1017895480

Reply:
Nav API team mentioned earlier today that these changes are not yet available. Unless the situation is different @dzinad?

Reply:
When did they mention that?
It's working for me. See
`https://api.mapbox.com/directions/v5/mapbox/driving-traffic/138.4282743,34.9665031;138.42594,34.9665461?access_token=<TOKEN>&geometries=polyline6&overview=full&steps=true&compute_toll_cost=true`
vs 
`https://api.mapbox.com/directions/v5/mapbox/driving-traffic/138.4282743,34.9665031;138.42594,34.9665461?access_token=<TOKEN>&geometries=polyline6&overview=full&steps=true&compute_toll_cost=true&waypoints_per_route=true`.

Or do you mean EV waypoints? Then probably yes, it's still in development but this parameter is not about EV. Anyways, I'm not against not deprecating it at all.